### PR TITLE
[SeoBundle] Fix hanging flash message, it will now disappear after save

### PR DESCRIPTION
**Pull request by denbatte**
The flash message did not go away after save.
Refactored the code to solve this issue

| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Fixed tickets | /

### CODE REVIEW

File: src/Kunstmaan/SeoBundle/Controller/Admin/SettingsController.php [hunk 1]
```diff
@@ -68,6 +51,11 @@ public function robotsSettingsAction(Request $request)
             }
         }
 
+        if ($robot->getRobotsTxt() === $default) {
```
Comment (jockri) [inline src/Kunstmaan/SeoBundle/Controller/Admin/SettingsController.php]: I think this is not the correct check. When someone saves the default text, there is an entry in the database, but `$robot->getRobotsTxt() === $default`is still true...

Comment (krispypen) [inline src/Kunstmaan/SeoBundle/Controller/Admin/SettingsController.php]: you should show the warning only when the value is null, and if the value is null, the default should be used in the "front"

**Comment by krispypen** [comment]
@denbatte what is the status of this pr?

**Comment by krispypen** [comment]
@roderik :+1:

**Pull request merged**